mozilla-release/mobile/android/app/src/main/res/menu-large/browser_app_menu.xml

```diff
@@ -78,7 +78,7 @@
 
             <!-- Title is overridden dynamically (it can also be for unpinning). -->
             <item android:id="@+id/pin_to_top_sites"
-                  android:title="@string/contextmenu_pin_to_top_sites"/>
+                  android:title="@string/cliqz_contextmenu_pin_to_top_sites"/>
```

Comment:
@spacifici for unpinning it should be `cliqz_contextmenu_unpin_from_top_sites`, but as stated in the comment, title is overridden dynamically, so my guess is that it will be replaced with `contextmenu_unpin_from_top_sites`

Reply:
I checked and it should be fine. We are handling the title dynamically in BrowserApp.java

Reply:
:+1: